samples/then.ts.scala

```diff
@@ -0,0 +1,22 @@
+
+import scala.scalajs.js
+import js.annotation._
+import js.|
+
+package importedjs {
+
+package `then` {
+
+@js.native
+trait Thenable[T] extends js.Object {
+  def `then`[TResult](onfulfilled: js.Function1[T, TResult | Thenable[TResult]] = ???, onrejected: js.Function1[js.Any, TResult | Thenable[TResult]] = ???): Thenable[TResult] = js.native
+}
+
+@js.native
+@JSGlobal("then.`then`")
```

Comment:
```
then.`then`
```
looks like a bug?

Reply:
Yes, that's a bug. It should be `"then.then"`